module/project/templates/helper.js

```diff
@@ -1077,7 +1077,8 @@ module.exports = class ProjectTemplatesHelper {
     
                     let project = await projectQueries.projectDocument({
                         userId : userId,
-                        projectTemplateId : templateData[0]._id
+                        projectTemplateId : templateData[0]._id,
+                        isAPrivateProgram : isAPrivateProgram
```

Comment:
@ankitpws I think the project template id will be different for pvt and public project, not sure if this condition makes sense.